Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                        "votes": 1024
+                    }, {
+                        "answer": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "answer": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
             }
+        ]
+```
+
+> **Note:** API Blueprint is **indentation sensitive**. Assets have to be **indented by four spaces** relative to it's existing indentation level. In the case of list items or sections, which are already indented. It results in having you include additional indentation, this further makes it a total of **eight spaces** when using assets inside a list item.
 
-    + Response 200
+In our API, the polls resource has a second action allowing you to create a new question. Inside our action, we've added a description which describes the structure you would send to the server to create a new question.
 
-        [Gist][]
+```markdown
+### Create a new question [POST]
 
-    ### Delete a Gist [DELETE]
-    + Response 204
+You can create your own question using this action. It takes a JSON dictionary containing a question and a collection of answers in the form of choices.
 
-Here are a few new things to learn about payloads:
+- question (string) - The question
+- choices (array[string]) - A collection of choices.
+```
 
-#### Request and its Payload
+We can show an example request, in this case we're showing an example JSON request you can make to the API.
 
-    + Request (application/json)
+```markdown
++ Request (application/json)
 
             {
-                "content": "Updated file contents"
+                "question": "Favourite programming language?",
+                "choices": [
+                    "Swift",
+                    "Python",
+                    "Objective-C",
+                    "Ruby"
+                ]
             }
+```
 
-Our "Edit a Gist" action needs to receive some input data to update the state of the Gist resource with it. The data are part of an HTTP request message. This section defines such a Request and an example [payload][] of the request message.
-
-> **NOTE:** The Request follows exactly the same structure as a Response or Model. As with the Resource Model a Request may not include a response status code.
-
-#### Empty Payload
-
-    ### Delete a Gist [DELETE]
-    + Response 204
-
-In the case of the "Delete a Gist" action our response bears status code "204" indicating the server has successfully fulfilled the request and that there is no additional content to be send in the response payload body.
+This example request will result in a 201 response from the server specifying we've created a new resource. The API returns a `Location` header specifying the URI for the created resource along with a JSON body.
```

Comment:
Reading this one loud does not feel right. Probably the "specifying" should be replaced. Or, better, do not teach HTTP here and skip explaining what 201 means.